chingu-voyages/v51-tier3-team-33
# Olga/feature/friends and groups display

# Description
Created pages to view all user friends and all user groups and be able to delete them


## Main changes explained:
- Created UI components to view friends and groups when clicking 'friends' and 'groups' links on the sidebar
- Added delete handlers to delete friends and groups
- Added success and error toasts to confirm deletion


## How to test:
1. check into current branch
2. do `pnpm install` and `pnpm run dev` to run this PR locally
3. login with a user who has friends and groups
4. go to `/home` 
5. click 'friends' and 'groups' on the sidebar
6. verify that you see user's groups and friends and can delete them
7. To view changes in the sidebar, you need to refresh the page

## Screenshots or videos of changes:

Friends:
![Screenshot 2024-10-14 at 11 53 16 AM](https://github.com/user-attachments/assets/a9cb0001-07b3-4f78-9a61-412ec4e0877b)

Groups:
![Screenshot 2024-10-14 at 11 53 03 AM](https://github.com/user-attachments/assets/0397f5c4-bba6-4af7-befe-aac9d261b8c0)


## Note:
Delete functionality for groups is not complete yet as I can not find the delete api route
